test/exchange/ExchangeAccessFacet.test.ts

```diff
@@ -240,4 +240,68 @@ describe('ExchangeAccessFacet', () => {
 
     it('removes the FAST contract from the list of Fast members');
   });
+
+  describe('isMemberActive', async () => {
+    beforeEach(async () => {
+      // Deactivate Alice.
+      await spcMemberAccess.deactivateMember(alice.address);
+    });
+
+    it('returns true when a member is active', async () => {
+      const subject = await access.isMemberActive(bob.address);
+      expect(subject).to.eq(true);
+    });
+
+    it('returns false when a member is deactived', async () => {
+      const subject = await access.isMemberActive(alice.address);
+      expect(subject).to.eq(false);
+    });
+  });
+
+  describe('deactivateMember', async () => {
+    it('requires the caller to be an SPC member', async () => {
+      const subject = access.deactivateMember(alice.address);
+      await expect(subject).to.be
+        .revertedWith(REQUIRES_SPC_MEMBERSHIP);
+    });
+
```

Comment:
In addition with the standard `requires the caller to be an SPC member` test, I started adding tests that ensure that the SPC contract `isMember` method is called (via a mock). I don't know if it's overkill.